jaraco/develop/projects-run.py

```diff
@@ -2,40 +2,37 @@
 Routine to run a command across all projects.
 """
 
-import argparse
+import functools
 import subprocess
+from typing import List, cast
 
-import autocommand
+import typer
+from jaraco.ui.main import main
+from typing_extensions import Annotated
 
 from . import filters, git
 
-parser = argparse.ArgumentParser()
-parser.add_argument(
-    '--keyword',
-    '-k',
-    dest='selectors',
-    type=filters.Keyword,
-    default=filters.Selectors(),
-    action='append',
-)
-parser.add_argument(
-    '--tag',
-    '-t',
-    dest='selectors',
-    type=filters.Tag,
-    default=filters.Selectors(),
-    action='append',
-)
-parser.add_argument('args', nargs='*')
 
-
-@autocommand.autocommand(__name__, parser=parser)
-def main(
-    selectors: filters.Selectors,
-    args=None,
+@functools.partial(
+    main,
+    app=typer.Typer(
+        context_settings=dict(allow_extra_args=True, ignore_unknown_options=True)
+    ),
+)
+def run(
+    tag: Annotated[
+        List[filters.Tag], typer.Option('--tag', '-t', parser=filters.Tag)
```

Comment:
In these moments looking at usages of [deprecated aliases](https://docs.python.org/3/library/typing.html#deprecated-aliases) or [overly verbose ones](https://typing.readthedocs.io/en/latest/reference/best_practices.html#shorthand-syntax), I'm really unhappy that https://github.com/fastapi/typer/discussions/823 was never answered.
I reached out to Sebastián, but received no response.

Reply:
:(
That's disappointing. I imagine he's burned out. I can see it in the Discussion template, which requires community engagement on other issues to file a question.